storage/src/cache/filecache/cache_entry.rs

```diff
@@ -221,6 +221,15 @@ impl BlobCache for FileCacheEntry {
         bios: &[BlobIoDesc],
     ) -> StorageResult<usize> {
         let mut bios = bios.to_vec();
+        bios.iter_mut().for_each(|b| {
+            if let Some(ref chunks_meta) = self.meta {
+                // TODO: the first blob backend io triggers chunks array download.
+                if let BlobIoChunk::Address(_blob_index, chunk_index) = b.chunkinfo {
```

Comment:
Not related to this PR, but it looks a bit wired to construct `BlobIoChunk::Address` and convert it to `BlobIoChunk::Base` for v6 IO desc. Looks like we can just create `BlobIoChunk::Base` and remove `BlobIoChunk::Address`. @jiangliu wdyt?